### PR TITLE
Seed collections from flat dirs of images in a data dir; use rake to …

### LINKED ISSUE #83: re-evaluate our seed map image

### LINKED ISSUE #158: Create rake command to pre-seed image collections based on a local folder

### PR DESCRIPTION
**Pull request by csbailey5t**
…calculate histograms and palettes

Fixes #158 
Fixes #83 

Thanks to @versae for the pyenv fix for Travis.

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,14 +5,19 @@
...
-Dir.glob(image_files)
-  .reject{ |f| f[/.*_tmp.*/] }
-  .map { |path| Image.from_file_path(path) }
-
-collection = Collection.find_or_create_by!(name: 'Default Collection')
-collection.images = Image.all
-collection.save!
-collection.create_composite_histogram
-collection.create_palette
+Dir.chdir("data/")
```
Comment (mejackreed) [inline db/seeds.rb]: This might be problematic. You may want to consider `Rails.root.join(Settings.IMAGE_PATH)` or something like that.

Comment (csbailey5t) [reply db/seeds.rb]: I knew that this was probably bad Ruby code. I'll rework it.

Comment (csbailey5t) [reply db/seeds.rb]: @mejackreed Is this (https://github.com/sul-cidr/histonets/pull/255/commits/c2343cf8eff580457c3b6dd52b4435d73a7608f9#diff-f7a25bc30a409fc77050703fc048ca8cR9) better or did you think I should completely avoid using `Dir.chdir()`?

Comment (mejackreed) [reply db/seeds.rb]: @csbailey5t Yeah I think that's better. I don't use it (perhaps just a stylistic thing though). I prefer not to have to think about where the current context of the directory is if possible. Potentially could lead to human mistakes? (I don't know.. if it works its probably fine)

Comment (mejackreed) [reply db/seeds.rb]: Also have you considered that we put the fast large storage in a different location on the server? Do we also need to have that configurable here too?

Comment (csbailey5t) [reply db/seeds.rb]: I haven't.  I think that this is ultimately a stopgap measure to let us move forward with creating real collections and images from real data. Given that we need to figure out a real way of doing file IO for the whole app, with all the new data files we're writing and such, I think we'll definitely be revising this in the next work cycle.

**Review by versae — APPROVED** [review]

**Pull request merged**